Makefile

```diff
@@ -39,11 +40,28 @@ lint:
 		exit 1; \
 	fi
 
+.PHONY: ssvsigner-lint
+ssvsigner-lint:
+	cd ssvsigner && $(SSVSIGNER_RUN_TOOL) golangci-lint -c ../.golangci.yaml run -v ./... && cd ..
+	@if [ ! -z "${UNFORMATTED}" ]; then \
+		echo "Some files requires formatting, please run 'go fmt ./...'"; \
+		exit 1; \
+	fi
+
 .PHONY: full-test
 full-test:
 	@echo "Running all tests"
 	@go test -tags blst_enabled -timeout 20m ${COV_CMD} -p 1 -v ./...
 
+.PHONY: ssvsigner-test
+ssvsigner-test:
+	@echo "Running ssv-signer unit tests"
+	@cd ssvsigner && go test -tags blst_enabled -timeout 20m -race -covermode=atomic -coverprofile=coverage.out -p 1 ./... && cd ..
+
+.PHONY: unit-test-with-signer
+unit-test-with-signer: unit-test ssvsigner-test
+	@echo "Running all tests including ssv-signer"
```

Comment:
IMO this target is redundant. Instead, modify the `unit-test` target to include ssvsigner tests:
 ```makefile
  .PHONY: unit-test
  unit-test:
        @echo "Running unit tests"
        @go test -tags blst_enabled -timeout 20m -race -covermode=atomic -coverprofile=coverage.out -p 1 `go list ./... | grep -ve "spectest\|integration\|ssv/scripts/"`
        @echo "Running ssv-signer unit tests"
        @$(MAKE) ssvsigner-test
```
Then remove this target entirely.